src/components/InlineAlert/InlineAlert.mdx

```diff
@@ -43,3 +44,25 @@ A message from the system to the user that offers contextual and timely informat
     'If you are promoting, soliciting feedback, or informing the user about a feature, use a banner instead.',
   ]}
 />
+
+<SubsectionHeader title="Variants" />
+
+### Neutral
+
+<Canvas of={InlineAlertStories.Neutral} />
+
+### Informational
+
+<Canvas of={InlineAlertStories.Informational} />
+
+### Error
+
+<Canvas of={InlineAlertStories.Error} />
+
+### Warning
+
+<Canvas of={InlineAlertStories.Warning} />
+
+### Success
+
+<Canvas of={InlineAlertStories.Success} />
```

Comment:
optionally you can add the playground as a story to be visible in the overview page